src/knapsack/SimpleKnapsackModel.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2020, Battelle Energy Alliance, LLC
+# ALL RIGHTS RESERVED
+"""
+Created on February 2, 2021
+
+@author: mandd
+"""
+
+#External Modules---------------------------------------------------------------
+
+#External Modules End-----------------------------------------------------------
+
+#Internal Modules---------------------------------------------------------------
+from PluginsBaseClasses.ExternalModelPluginBase import ExternalModelPluginBase
+from utils import InputData, InputTypes
+#Internal Modules End-----------------------------------------------------------
+
+
+class SimpleKnapsackModel(ExternalModelPluginBase):
```

Comment:
This class needs to be updated with the base class.

Reply:
yes, I'll updated the two derived classes